redwood-compose/src/commonTest/kotlin/app/cash/redwood/compose/ListeningTestRow.kt

```diff
@@ -47,6 +47,9 @@ class ListeningTestRow : TestRow<WidgetValue>, ChangeListener {
 
     override fun onModifierUpdated(index: Int, widget: Widget<WidgetValue>) {
     }
+
+    override fun detach() {
```

Comment:
Not doing any detach work for Compose UI until we need it. Especially since it doesn’t have a separate list for its views!